homeassistant/components/opengarage/cover.py

```diff
@@ -181,3 +129,22 @@ def device_class(self):
     def supported_features(self):
         """Flag supported features."""
         return SUPPORT_OPEN | SUPPORT_CLOSE
+
+    @property
+    def device_id(self):
+        """Return the ID of the physical device this sensor is part of."""
+        return self._device_id
+
+    @property
+    def unique_id(self):
+        """Return a unique ID."""
+        return f"{self.device_id}_cover"
```

Comment:
We don't need to add the platform domain to the unique_id. The entity registry is aware of both the integration domain and the platform domain.